### PR TITLE
e2e: continuing progress on ExitCleanly()

### PR DESCRIPTION
**Pull request by edsantiago**
Seven more e2e test files.

**DO NOT REVIEW THIS ENTIRE PR**. Review only the second commit.

```release-note
None
```

### CODE REVIEW

File: test/e2e/manifest_test.go [hunk 1]
```diff
@@ -192,14 +199,14 @@ var _ = Describe("Podman manifest", func() {
 
 		push := podmanTest.Podman([]string{"manifest", "push", "--all", "--add-compression", "zstd", "--tls-verify=false", "--remove-signatures", "foobar", "localhost:5000/list"})
 		push.WaitWithDefaultTimeout()
-		Expect(push).Should(ExitCleanly())
+		Expect(push).Should(Exit(0))
 		output := push.ErrorToString()
 		// 4 images must be pushed two for gzip and two for zstd
 		Expect(output).To(ContainSubstring("Copying 4 images generated from 2 images in list"))
 
 		session = podmanTest.Podman([]string{"run", "--rm", "--net", "host", "quay.io/skopeo/stable", "inspect", "--tls-verify=false", "--raw", "docker://localhost:5000/list:latest"})
```
Comment (edsantiago) [inline test/e2e/manifest_test.go:207]: Hey so skopeo is available on all CI machines, it's a requirement, do we really have to pull the containerized version?

Comment (mheon) [reply test/e2e/manifest_test.go:207]: It adds the requirement of having a working Skopeo to run the tests locally, but that doesn't seem particularly egregious

**Comment by mheon** [comment]
LGTM

**Pull request merged**